telemetry/definitions/commonDefinitions.json

```diff
@@ -2609,50 +2589,74 @@
-                    "type": "awsqFilesChangedCount"
-                },
-                {
-                    "type": "awsqThumbsUpCount"
-                },
-                {
-                    "type": "awsqThumbsDownCount"
-                },
-                {
-                    "type": "awsqIsApproachAccepted"
-                },
-                {
-                    "type": "awsqServiceErrorsCount"
-                }
-            ]
+            "name": "awsq_assignCommand",
+            "description": "User have triggered /assign",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqAssignCommandCount" }]
+        },
+        {
+            "name": "awsq_codeReGeneration",
+            "description": "User invoked the weaverbird agent to re-engage in Code gen",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqCodeReGenerationCount" }]
+        },
+        {
+            "name": "awsq_codeGenerateClick",
+            "description": "User has clicked on Generate Code after have iterated through the Approach phase",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqGenerateCodeClickCount" }]
+        },
+        {
+            "name": "awsq_approach",
+            "description": "User received an Approach",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqApproachGeneratedCount" }]
+        },
+        {
+            "name": "awsq_isRegeneratedCode",
+            "description": "User has re-generated code as a part of code generation portion",
+            "metadata": [{ "type": "enabled" }]
+        },
+        {
+            "name": "awsq_filesReviewed",
+            "description": "User reviewed X files after Code Generation phase",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqFilesReviewedCount" }] 
+        },
+        {
+            "name": "awsq_isAcceptedCodeChanges",
+            "description": "User applied code changes generated for the task",
+            "metadata": [{ "type": "enabled" }] 
+        },
+        {
+            "name": "awsq_repo",
+            "description": "User provides awsq with repository",
+            "metadata": [{ "type": "awsqRepositorySize" }] 
+        },
+        {
+            "name": "awsq_filesChanged",
+            "description": "Awsq service suggested X files to change",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqFilesChangedCount" }] 
+        },
+        {
+            "name": "awsq_thumbsUp",
+            "description": "User clicked on the thumbs up button, to mention that they are satisfied",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqThumbsUpCount" }] 
+        },
+        {
+            "name": "awsq_thumbsDown",
+            "description": "User clicked on the thumbs down button to say that they are unsatisfied",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqThumbsDownCount" }]
+        },
```

Comment:
in all of these cases, the `unit: count` declaration means the value can be set on the metric without the need for a special-purpose field (awsqFilesChangedCount, awsqThumbsUpCount, ...).

Please look at existing examples in this file, as well as usages in the Toolkit, to understand how things can be shaped.

For example, `aws_loadCredentials` has `unit:Count` : https://github.com/aws/aws-toolkit-common/blob/1c935af73d9042acf10c44e21f5f23b3895ae619/telemetry/definitions/commonDefinitions.json#L1075
and this is how it's used in the Toolkit (`value: 1`): https://github.com/aws/aws-toolkit-vscode/blob/61737287d918f777ea2e4a389ffea08fcc950adc/src/auth/providers/credentialsProviderManager.ts#L32

Reply:
from the example give it is **not** clear that the metadata value can be omitted.
```
 telemetry.aws_loadCredentials.emit({ credentialSourceId: telemType, value: 1 })
 ```
 it seems like it increments on credentialSourceId.
 I will remove Count values, tho